tools/pythonpkg/tests/fast/api/test_read_csv.py

```diff
@@ -544,3 +544,23 @@ def test_read_csv_names_mixed_with_dtypes(self):
                     'f': str,
                 },
             )
+
+    def test_read_csv_multi_file(self):
+        con = duckdb.connect()
+        file1 = StringIO('one,two,three,four\n1,2,3,4\n1,2,3,4\n1,2,3,4')
+        file2 = StringIO('one,two,three,four\n5,6,7,8\n5,6,7,8\n5,6,7,8')
+        file3 = StringIO('one,two,three,four\n9,10,11,12\n9,10,11,12\n9,10,11,12')
+        files = [file1, file2, file3]
+        rel = con.read_csv(files)
+        res = rel.fetchall()
```

Comment:
Can you also test this with a file that doesn't exist in the middle of the list?